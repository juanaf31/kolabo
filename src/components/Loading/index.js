import { Spin } from 'antd';

const Loading = () => {
	return (
		<div className="container-spinner">
			<Spin tip="Loading..." size="large" />
		</div>
	);
};

export default Loading;
