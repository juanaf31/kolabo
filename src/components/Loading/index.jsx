import { Spin } from 'antd';

const Loading = () => {
	return (
		<div data-testid='loading' className="container-spinner">
			<Spin tip="Loading..." size="large" />
		</div>
	);
};

export default Loading;
