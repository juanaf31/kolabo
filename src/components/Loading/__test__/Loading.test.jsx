import { cleanup, render } from '@testing-library/react'
import React from 'react'
import Loading from '../index'
import renderer from 'react-test-renderer'

afterEach(() => {
    cleanup()
})

describe('testing card component', () => {

    test('render components correctly', () => {
        const { getByTestId } = render(<Loading />)
        getByTestId('loading')
    })

    test('loading matches snapshot', () => {
        const tree = renderer.create(<Loading />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})