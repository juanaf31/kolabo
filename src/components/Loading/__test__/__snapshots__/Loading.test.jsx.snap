// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing card component loading matches snapshot 1`] = `
<div
  className="container-spinner"
  data-testid="loading"
>
  <div
    className="ant-spin ant-spin-lg ant-spin-spinning ant-spin-show-text"
  >
    <span
      className="ant-spin-dot ant-spin-dot-spin"
    >
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
    </span>
    <div
      className="ant-spin-text"
    >
      Loading...
    </div>
  </div>
</div>
`;
